app/src/main/java/com/example/tasky/feature_agenda/presentation/util/UsernameUtil.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.tasky.feature_agenda.presentation.util
+
+fun getInitials(value: String): String {
+    val names = value.split(Regex("\\s+"))
```

Comment:
I'd also filter out empty strings here. If the name is `Philipp  Lackner` (2 spaces), split will lead to a list like this: `["Philipp", "", "Lackner"]`. That would lead to an index out of bounds error below